app/lib/features/activities/pages/activities_page.dart

```diff
@@ -41,36 +41,30 @@ class ActivitiesPage extends ConsumerWidget {
         } else if (sessions.length > 1) {
           children.add(
             SliverToBoxAdapter(
-              child: Padding(
-                padding: const EdgeInsetsDirectional.symmetric(
-                  horizontal: 10,
-                  vertical: 5,
-                ),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: [
-                    Text(
-                      'You have ${sessions.length} unverified sessions log in',
-                      style: Theme.of(context).textTheme.headlineSmall,
-                    ),
-                    ElevatedButton(
-                      onPressed: () {
-                        context.pushNamed(Routes.settingSessions.name);
-                      },
-                      style: ElevatedButton.styleFrom(
-                        backgroundColor: Theme.of(context).colorScheme.neutral,
-                        shape: RoundedRectangleBorder(
-                          borderRadius: BorderRadius.circular(6),
-                        ),
-                        side: BorderSide(
-                          color: Theme.of(context).colorScheme.success,
-                        ),
-                        foregroundColor: Theme.of(context).colorScheme.neutral6,
-                        textStyle: Theme.of(context).textTheme.bodySmall,
+              child: Card(
+                child: ListTile(
+                  leading: const Icon(Atlas.warning_bold),
+                  title: Text(
+                    'There are ${sessions.length} unverified sessions logged in',
+                    style: Theme.of(context).textTheme.headlineSmall,
+                  ),
+                  trailing: ElevatedButton(
+                    onPressed: () {
+                      context.pushNamed(Routes.settingSessions.name);
+                    },
+                    style: ElevatedButton.styleFrom(
+                      backgroundColor: Theme.of(context).colorScheme.neutral,
+                      shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(6),
+                      ),
+                      side: BorderSide(
+                        color: Theme.of(context).colorScheme.success,
                       ),
-                      child: const Text('Review'),
+                      foregroundColor: Theme.of(context).colorScheme.neutral6,
+                      textStyle: Theme.of(context).textTheme.bodySmall,
                     ),
-                  ],
+                    child: const Text('Review'),
```

Comment:
I would like to keep it with context styling (see above line) for consistency in text size factor, colors etc.